cosmos/keybase.go

```diff
@@ -16,9 +17,9 @@ const mnemonicEntropySize = 256
 
 // Keybase is a standard cosmos keybase.
 type Keybase struct {
-	kb keys.Keybase
-
-	mx sync.Mutex
+	kb        keys.Keybase
+	mx        sync.Mutex
+	privCache map[[sha256.Size]byte]crypto.PrivKey
```

Comment:
```suggestion
	privKeyCache map[[sha256.Size]byte]crypto.PrivKey
```

or just `privKeys`, but privCache expands to private cache 😕